src/upgrade/versions/versions.php

```diff
@@ -52,14 +52,13 @@
 $versions['18.02.0'] = array(
     'to' => '20.0.0',
     // Use older update to get new field that was missed in previous "new install" SQL snapshot
-    'folder' => '17.12.0',
+    'folder' => '17.12.0'
```

Comment:
No need to change it since it is minor, I just wanted to note for the future about using comma on the last item of an array: it is a good idea to use one for the sake of slightly easier to follow git history.  

That way when a new entry is added to the end of the array later, the git history only shows the new line as the one changed.  If you have to add a comma on the previous line as part of your change then the git history would show both lines as changed.  It is a minor detail but having the end comma on the last item of an array makes things easier on git history.

And actually it would affect this particular PR:  Now it shows all the `'folder' => ...` lines as changed even though that wasn't really related to your changes of removing the changelog line.  If you kept the comma on the line the history won't show your commit as changing it.

Reply:
I wasn't sure the best approach. The issue that bother my OCD was that some had a trailing comment and others did not. So, I wanted to be consistent and chose the wrong option.

Reply:
> I wasn't sure the best approach. The issue that bothered my OCD was that some had a trailing comma and others did not. So, I wanted to be consistent and chose the wrong option.

Heh nothing wrong with wanting consistency!

I have to resist the temptation to change the trailing comma myself.  And sometimes I forget, or maybe it will be on a single line and I'll change it to 1 line per array entry and forget to put the last one.  But if I did go adding the comma when it is missed that kind of defeats the purpose for having easier to read git history so then I can't give that as my "reason" for doing so 😆 .  Usually I don't change an existing one unless I'm already adding or removing from it anyways.

I think maybe we can set up a PHPCS rule to require trailing comma, then it would all be done in one go, and once we have `cs-check` able to run with no errors, it would be part of the "standards sniffing" so it would complain if you try to commit something that was inconsistent.